tests/incremental/14-manual-restart/04-restarting_and_aborting.c

```diff
@@ -0,0 +1,16 @@
+int g = 0;
```

Comment:
Required after #845:
```suggestion
#include <goblint.h>

int g = 0;
```